website/views/pages/homepage.ejs

```diff
@@ -958,6 +959,10 @@
               <div purpose="comparison-column" v-else-if="comparisonModeForSecurity === 'qualys'">
                 <p>Mobile not supported</p>
               </div>
+              <div purpose="comparison-column" v-else-if="comparisonModeForSecurity === 'defender'">
```

Comment:
@mike-j-thomas maybe we should move this up to line:956? 

When looking at the defender learn site ...they do not explicitly state that Microsoft Defender for Endpoint, instead offering built-in, direct assessment against CIS benchmarks at all levels for its supported operating systems. Or is this cut corners too much?